### PR TITLE
Added SwiftyCDN

### PR DESCRIPTION
**Pull request by Zlopi**
Add new CDN provider SwiftyCDN

Check please:
globaldialog.ru (only CNAME cdn.globaldialog.ru)
traceroute.net.ru (Primary)
melbicom.net  and melbicom.ru (Primary)

Thanks

### CODE REVIEW

File: internal/optimization_checks.py [hunk 1]
```diff
@@ -195,7 +195,8 @@ def __init__(self, job, task, requests):
                       '.yimg.',
                       '.yahooapis.com'],
             'Yottaa': ['.yottaa.net'],
-            'Zenedge': ['.zenedge.net']
+            'Zenedge': ['.zenedge.net'],
+            'SwiftyCDN': ['.swiftycdn.net']
```
Comment (pmeenan) [inline internal/optimization_checks.py]: Please put these in alphabetical order.

Comment (Zlopi) [reply internal/optimization_checks.py]: Done

**Comment by Zlopi** [comment]
I beg your pardon, the US-connectivity is still being optimized (it should be better today), but you're right - SwiftyCDN has more european PoPs (please see the connectivity details at https://www.host-tracker.com/InstantCheck/3/0769be4e-ccae-e911-a409-0003ff7328cc/ and https://www.host-tracker.com/InstantCheck/2/19f13786-ccae-e911-a409-0003ff7328cc/ for example). The site should also be ready soon.

**Comment by pmeenan** [comment]
Perfect, thank you.

**Comment by Zlopi** [comment]
Thank you. But now we have a problem on detecting the CDN on your site: melbicom.net is being detected as "Myra Security CDN'. Should we probably update the header to something more specific (while we still can do so)?

> CDN's Used:
> melbicom.net : Myra Security CDN, SwiftyCDN

**Pull request merged**